client/src/stores/diagramStore.ts

```
@@ -1,1321 +1,1321 @@
+      const snappedPosition = snapToGridPosition(position, gridSize, snapToGrid);
+      const id = `${type}-${Date.now()}`;
+      let newNode: Node;
+
+      switch (type) {
+        case 'event':
+          const eventData: EventNodeData = { 
+            id, 
+            nodeType: 'event', 
+            eventType: options.eventType || 'start', 
+            label: options.eventType === 'end' ? 'End' : options.eventType === 'intermediate' ? 'Intermediate' : 'Start'
+          };
```

Biome (2.4.16)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)